lib/src/pdf_page_view.dart

```diff
@@ -47,7 +53,8 @@ class PDFPageView {
         'renderer': rendererString,
         'annotationLayerFactory': annotationLayerFactory?._jsInternal,
         'textLayerFactory': textLayerFactory?._jsInternal,
-        'eventBus': JsObject(PDFJS.versionSafePdfJsViewerContext['EventBus'] as JsFunction),
+        'eventBus': JsObject(context['pdfjsViewer']['EventBus'] as JsFunction),
+        'annotationMode': annotationMode == AnnotationMode.disabled ? 0 : 2,
```

Comment:
From what I can tell, the [default value](https://github.com/Workiva/pdfjs_cdn/pull/2/files?authenticity_token=XW5WVFE03k7bHft3lLpJVHasYvCexTXfELm5CtTPwadw-0Ftg3K0Y00ljSRKEZYEagN_umLkC43uCaTaVgE6aQ&file-filters%5B%5D=#diff-cc454e757ffec1e6487805e88c99b0a3632feff069a0c984dba7970d735b86d6R4687) appears to be 2 
<img width="799" alt="Screen Shot 2022-01-07 at 3 28 15 PM" src="https://user-images.githubusercontent.com/76955103/148609729-5ed322e5-59ea-432c-ba5f-5e160a9fd557.png">

```
const AnnotationMode = {
  DISABLE: 0,
  ENABLE: 1,
  ENABLE_FORMS: 2,
  ENABLE_STORAGE: 3,
};
```

Reply:
Still need to check the behavior on if we would pass this option into an older version of pdf.js. Hopefully it would ignore it, but 🤷

Reply:
Smoke tested with older versions of pdf.js (2.5.207), and didn't see any abnormalities in using this